### ISSUE TITLE
Remove unnecessary aria-labels

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5192]
The aria-labels are unnecessary in a couple places because the landmark role sufficiently describes them. Remove the `aria-label` attributes in:

- [ ] the sidebar
- [ ] the breadcrumbs

**Comment by israelcefrin** [comment]
@NateWr removing the `aria-label` from the Sidebar will remove a translation key (`common.navigation.sidebar`) that doesn't seem to be used in any other part of the front-end or the backend. Maybe we should create an issue to review whether other themes are using this key and double-check if we should keep it on the locale files.

**Comment by NateWr** [comment]
I found it is used in the `bootstrap3` and `healthSciences` themes.

**Comment by israelcefrin** [comment]
I think we can keep the locale key for now. Given these themes has the same issue to fix, I will open PRs for each of them after fixing the issues for the Default Theme. Once all themes were not using this key anymore we can review how to remove it from the locale files since it will not be used anywhere else.

**Comment by israelcefrin** [comment]
I've opened a PR to adjusting it: #8108 
Please, let me know if everything was made correctly (code and PR workflow)

**Comment by NateWr** [comment]
Looks good @israelcefrin. Can you go ahead and create a submodule commit to run the tests in OJS? You can read about how to do that here: https://docs.pkp.sfu.ca/dev/testing/en/continuous-integration#prepare-pull-requests-for-testing

Then, once the tests have passed, let me know and I can merge it.

**Comment by israelcefrin** [comment]
Hi @NateWr 

The code passed the test and has no conflicts with the base branch. 

However, the build has failed  on Stage 2 : PHP 8.1 
https://app.travis-ci.com/github/pkp/ojs/jobs/577732646

It seems the failed test is related to the OJS code instead the theme though.

**Comment by NateWr** [comment]
It's probably ok, but I've restarted that test to be sure. Can you keep an eye out and let me know when it's passed?

**Comment by israelcefrin** [comment]
The test passed successfully now.
